src/geoengine/layers.md

```diff
@@ -0,0 +1,18 @@
+# Layers
+
+A layer is a browsable unit in Geo Engine.
+In general, it is a named [`Workflow`](../api/workflows.md) with additional meta information like a description and a default [`Colorizer`](../datatypes/colorizer.md).
```

Comment:
should link to "Workflows" in the geo engine chapter and not the API

Reply:
That is correct, but it is just not there, yet.
So I added a TODO:
[68eb7d6](https://github.com/geo-engine/docs/pull/44/commits/68eb7d69373835284023e7f75171f98912b80ae5)